app/src/main/java/org/oppia/android/app/topic/lessons/StorySummaryViewModel.kt

```diff
@@ -42,13 +42,11 @@ class StorySummaryViewModel(
   }
 
   fun computeStoryNameChapterCountContainerContentDescription(): String {
-    // TODO(#3844): Combine these strings together.
-    val chapterCountText =
-      resourceHandler.getQuantityStringInLocaleWithWrapping(
-        R.plurals.chapter_count, storySummary.chapterCount, storySummary.chapterCount.toString()
-      )
-    return resourceHandler.getStringInLocaleWithWrapping(
-      R.string.chapter_count_with_story_name, chapterCountText, storyTitle
+    return resourceHandler.getQuantityStringInLocaleWithWrapping(
```

Comment:
Are there tests to verify the two cases?

Reply:
Not yet. Let me work on those.